docs/source/user_guide/automl.ipynb

```diff
@@ -645,17 +1598,1393 @@
+      "\n",
+      "Using DaskEngine to train and score pipelines.\n",
+      "Searching up to 1 batches for a total of 3 pipelines. \n",
+      "Allowed model families: linear_model\n",
+      "\n",
+      "Evaluating Baseline Pipeline: Mode Baseline Binary Classification Pipeline\n",
+      "Mode Baseline Binary Classification Pipeline:\n",
+      "\tStarting cross validation\n",
+      "\tFinished cross validation - mean Log Loss Binary: 12.868\n",
+      "\n",
+      "*****************************\n",
+      "* Evaluating Batch Number 1 *\n",
+      "*****************************\n",
+      "\n",
+      "Elastic Net Classifier w/ Imputer + Standard Scaler:\n",
+      "\tStarting cross validation\n",
+      "\tFinished cross validation - mean Log Loss Binary: 0.077\n",
+      "Logistic Regression Classifier w/ Imputer + Standard Scaler:\n",
+      "\tStarting cross validation\n",
+      "\tFinished cross validation - mean Log Loss Binary: 0.077\n",
+      "\tHigh coefficient of variation (cv >= 0.2) within cross validation scores.\n",
+      "\tLogistic Regression Classifier w/ Imputer + Standard Scaler may not perform as estimated on unseen data.\n",
+      "\n",
+      "Search finished after 00:04            \n",
+      "Best pipeline: Logistic Regression Classifier w/ Imputer + Standard Scaler\n",
+      "Best pipeline Log Loss Binary: 0.076807\n"
+     ]
+    }
+   ],
+   "source": [
+    "dask_engine_p4 = DaskEngine(Client(LocalCluster(processes=False, n_workers = 4)))\n",
+    "\n",
+    "automl_dask_p4 = AutoMLSearch(X_train=X, y_train=y,\n",
+    "                      problem_type=\"binary\",\n",
+    "                      allowed_model_families=[ModelFamily.LINEAR_MODEL],\n",
+    "                      engine = dask_engine_p4)\n",
+    "automl_dask_p4.search(show_iteration_plot = False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As we can see, a significant performance gain can result from simply using something other than the default `SequentialEngine`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 31,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2021-07-27T18:56:35.397855Z",
+     "start_time": "2021-07-27T18:56:35.394488Z"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Sequential search duration: 19.669402837753296\n",
+      "Concurrent futures (threaded) search duration: 3.979705810546875\n",
+      "Dask (two threads) search duration: 9.009714126586914\n",
+      "Dask (four processes)search duration: 4.219135046005249\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(\"Sequential search duration: %s\" % str(automl.search_duration))\n",
+    "print(\"Concurrent futures (threaded) search duration: %s\" % str(automl_cf_threaded.search_duration))\n",
+    "# print(\"Concurrent futures (process) search duration: %s\" % str(automl_cf_process.search_duration))\n",
```

Comment:
nit: It kinda looks awk to have this one commented out. I think you can remove since you've gotten the point across regardless